lib/tasks/assemble.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 const chalk = require('chalk');
-const install = require('rsvp').denodeify(require('npmi'));
+const pruner = require('rsvp').denodeify(require('electron-packager/prune').pruneModules);
```

Comment:
Very fair, I felt uncomfortable doing it as well. Wanted to show intent and happy to make any changes.